src/components/Tender.vue

```diff
@@ -0,0 +1,488 @@
+  </v-card>
+
+  <v-container style="width: 67vw;">
+    <v-row class="mt-3 mb-10 justify-end">
+      <v-col cols="12" md="2"><v-btn type="submit" block class="mt-2" variant="outlined" color="blue"
+          size="large">Cancel</v-btn></v-col>
+      <v-col cols="12" md="2"><v-btn type="submit" block class="mt-2" variant="flat" color="blue"
+          size="large">Publish</v-btn></v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import SvgIcon from "@jamescoyle/vue-icon";
+import { mdiInformationOutline } from "@mdi/js";
+import { countries } from "@/components/countries";
+import { tenderTypes } from "@/components/tenderTypes";
+import { currencies } from "@/components/currencies";
+
+export default {
+  components: {
+    SvgIcon
+  },
+  data: () => ({
+    countries: countries,
+    tenderTypes: tenderTypes,
+    currencies: currencies,
+
+    tender: {
+      organizationName: '',
+      nationalRegistrationNumber: '',
+      country: null,
+      city: '',
+      firstName: '',
+      lastName: '',
+      phone: '',
+
+      type: null,
+      description: '',
+      maxPrice: '',
+      minPrice: '',
+      currency: null,
+      publicationDate: new Date().toLocaleDateString(),
+      deadline: null,
+
+      deadlineForSignedContract: null,
+    },
+
+    path: mdiInformationOutline,
+    valid: false,
+    dialog: false,
+    dialog1: false,
+    dialog2: false,
+    selectedFile: null,
+    selectedContract: false,
+    contract: true,
+    awardDecision: true,
+    selectedAwardDecision: false,
+    rejectDecision: true,
+    selectedRejectDecision: false,
+    docFileUrl: '',
+    contractIsSelecting: false,
+    awardIsSelecting: false,
+    rejectIsSelecting: false,
+
+    selectedFileContract: null,
+    selectedFileContractName: '',
+    selectedFileContractUrl: '',
+    selectedFileAwardDecision: null,
+    selectedFileAwardDecisionName: '',
+    selectedFileAwardDecisionUrl: '',
+    selectedFileRejectDecision: null,
+    selectedFileRejectDecisionName: '',
+    selectedFileRejectDecisionUrl: '',
+  }),
+
+  methods: {
+    handleFileImport(selectedDocument) {
+      switch (selectedDocument) {
+        case 'contract':
+          this.contractIsSelecting = true;
+          window.addEventListener('focus', () => {
+            this.contractIsSelecting = false;
+            this.selectedContract = true;
+            this.contract = false;
+          }, { once: true });
+          this.$refs.contractUploader.click();
+          break;
+        case 'awardDecision':
+          this.awardIsSelecting = true;
+          window.addEventListener('focus', () => {
+            this.awardIsSelecting = false;
+            this.selectedAwardDecision = true;
+            this.awardDecision = false;
+          }, { once: true });
+          this.$refs.awardDesUploader.click();
+          break;
+        case 'rejectDecision':
+          this.rejectIsSelecting = true;
+          window.addEventListener('focus', () => {
```

Comment:
frameworks like Vue are dedicated to eliminate direct DOM manipulations like window.addEventListener. You should never use addEventListener or getElementById unless it it the only way to do smth